05_syntax_lab/06.py

```diff
@@ -6,4 +6,35 @@
 For example if the numbers were 4 and 6,
 program should print 12
 """
+# Exercise 6
+# By Zvi Figov
+from random import randint
+
+num1 = randint(1,10)
+num2 = randint(1,10)
+# brute force 
+max_m = num1*num2
+# get multipliers of num1
+mul1 = ""
+for i in range(num1,max_m+1,num1):
+    mul1 = mul1 + str(i) + "\n"
+# get multipliers of num2
+mul2 = ""
+for i in range(num2,max_m+1,num2):
+    mul2 = mul2 + str(i) + "\n"
+# get common multipliers
+common_mult = ""
+for i in mul1.split():
+    for j in mul2.split():
+        if i==j:
+            common_mult = common_mult + i + "\n"
+# get minimum common mult
+min_c = 999999
+for i in common_mult.split():
+    if int(i)<min_c :
+        min_c = int(i)
+print "Least common multiplier of %d and %d = %d" % (num1,num2,min_c)
+
+
+
 
```

Comment:
Using strings when we are dealing with numbers is quite confusing. Please don't do that.
I am not sure if generating all the multipliers of both numbers is efficient, but I think there are better solutions. Try to look for a faster algorithm. Especially think about the case when you have two large numbers to work with.